src/components/DepositFee/styles.ts

```diff
@@ -0,0 +1,272 @@
+  `}
+`
+
+type ButtonWrapperProps = {
+  showButtons: boolean
+}
+export const ButtonWrapper = styled.div<ButtonWrapperProps>`
+  ${() => css`
+    display: flex;
+    width: 100%;
+    gap: 2.4rem;
+
+    opacity: 0;
+    height: 0;
+    pointer-events: none;
+
+    transition-timing-function: ease-in;
+    transition-duration: 300ms;
+    transition-property: opacity height;
+  `}
+
+  ${({ showButtons }) =>
+    showButtons &&
+    css`
+      height: auto;
+      opacity: 1;
+      margin-top: 2.4rem;
+      pointer-events: all;
+    `}
+`
+
+export const DepositFeeHeader = styled.span`
+  ${({ theme }) => css`
+    display: flex;
+    justify-content: space-between;
+
+    width: 100%;
+
+    > h3 {
+      color: ${theme.colors.white};
+      font-weight: ${theme.font.weight.normal};
+      font-size: ${theme.font.sizes.font14};
+      line-height: 1.6rem;
+      letter-spacing: 0.22em;
+      text-transform: uppercase;
+    }
+  `}
+`
+
+export const CardWrapperParagraph = styled.p`
+  ${({ theme }) => css`
+    color: ${theme.colors.white};
+    font-weight: ${theme.font.weight.light};
+    font-size: ${theme.font.sizes.font16};
+    line-height: 135%;
+  `}
+`
+
+interface IFeeContainerProps {
+  isFeeChecked?: boolean
+}
+
+export const FeeContainer = styled.div<IFeeContainerProps>`
+  ${() => css`
+    overflow: hidden;
+
+    max-height: 0;
+    padding: 0.1rem;
+
+    opacity: 0;
+    pointer-events: none;
+
+    transition-timing-function: ease-out;
+    transition-duration: 700ms;
+    transition-property: max-height opacity;
+  `}
+  ${({ isFeeChecked = true }) =>
+    isFeeChecked &&
+    css`
+      max-height: 500px;
+
+      opacity: 1;
+      pointer-events: auto;
+
+      transition-timing-function: ease-in;
+      transition-duration: 700ms;
+      transition-property: max-height opacity;
+    `}
+`
+
+interface IWrapperInputFeeProps {
+  isAddress: boolean
+  value: number
+}
+
+export const WrapperInputFee = styled.span<IWrapperInputFeeProps>`
+  ${({ theme, isAddress }) => css`
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+
+    margin-top: 1.6rem;
+
+    ${Input}[type="number"]:invalid:not([value=""]) {
+      border: 1px solid ${theme.colors.error};
+    }
+    ${Input}[type="number"]:invalid:not([value='']) ~ ${Error} {
+      color: ${theme.colors.error};
+    }
+
+    ${Input}[type="text"]:valid {
+      border: ${isAddress
+        ? `1px solid ${theme.colors.success};`
+        : `1px solid ${theme.colors.error};`};
+    }
+
+    ${Input}[type="text"]:valid ~ ${Error} {
+      display: ${!isAddress && 'block'};
+    }
+  `}
+  ${({ theme, value }) =>
+    value > 50 &&
+    value <= 95 &&
+    css`
+      ${Input}[type="number"]:valid {
+        border: 1px solid ${theme.colors.amber};
+      }
+      ${Input}[type="number"]:not([value='']) ~ ${Error} {
+        display: block;
+
+        color: ${theme.colors.amber};
+      }
+    `}
+`
+
+export const RefferalCommissionWrapper = styled.div`
```

Comment:
It is spelled "referral"

```suggestion
export const ReferralCommissionWrapper = styled.div`
```